service/test/testdata/TestCreateProject/can_create_project_response.json

```diff
@@ -1,3 +1,4 @@
 {
-  "token": "vault::"
+  "token": "vault::",
+  "token_id": ""
```

Comment:
Would an empty token ID mean that create project failed somewhere in its flow? If so, I would expect this test to fail.

Reply:
sure, updated the mock to provide values here. now expecting:
`
{
  "token": "vault:role-id:secret",
  "token_id": "secret-id-accessor"
}
`